docker/autoscaler/run_autoscaler.py

```diff
@@ -0,0 +1,49 @@
+import logging
+
+from ray import ray_constants
+from ray._private.ray_logging import setup_component_logger
+from ray._private.services import get_node_ip_address
+from ray.autoscaler._private.monitor import Monitor
+import yaml
+
+
+AUTOSCALING_CONFIG_PATH = "/autoscaler/ray_bootstrap_config.yaml"
+AUTOSCALING_LOG_DIR = "/tmp/ray/session_latest/logs/"
```

Comment:
seems this is common path. is there's a const can be reused from `ray`?

Reply:
Might not be worth it.

There's `get_ray_tmp_dir()` in  `ray._private.utils` for "tmp/ray" and `SESSION_LATEST ` in `ray.node` for "session_latest". 
"logs" is hard-coded in `ray.node`.

Reply:
In theory the right thing to do here is to the same thing as in https://github.com/ray-project/ray/blob/30968a93585c84137587af3bf9719966abf492d9/python/ray/node.py#L259 to get the actual concrete logging dir (not `session_latest`, which is a symlink to the concrete logging dir). However it is not worth doing that here, it would either need a big refactor or an additional GCS client, and on K8s there will only ever be one concrete logging dir since the pods are not re-used between Ray cluster instantiations. So I'd like to keep it simple as it is right now.

I think the suggestion that @DmitriGekhtman is making here makes sense. That will keep it a bit more generic, in case we change the conventions later (e.g. the right place for logs is really `/var/log` and not `/tmp/`, but we can't use `/var/log` because then Ray would not run out of the box on non-root or restricted permissions environments). So I'll implement that.